documents/openapi/openapi.yaml

```diff
@@ -2512,6 +2547,21 @@ components:
       required:
         - recipient_identifier
         - template_id
+    PushNotificationBroadcastRequest:
+      type: object
+      properties:
+        mobile_app:
+          $ref: "#/components/schemas/MobileApp"
+        template_id:
+          $ref: "#/components/schemas/Id"
+        personalisation:
+          $ref: "#/components/schemas/Personalisation"
+        topic_sid:
+          type: string
+          example: 8358D9FB183646F3CD16982562530955
```

Comment:
Please change this to XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

Reply:
done